packages/builder/src/components/deploy/RevertModalVersionSelect.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
```

Comment:
This is the toggle for selecting arbitrary revert versions, should only be visible in development.